accelerator/core/apis.c

```diff
@@ -367,10 +377,9 @@ status_t api_recv_mam_message(const iota_config_t* const iconf, const iota_clien
   bundle_transactions_t* bundle = NULL;
   BUNDLE_ARRAY_FOREACH(bundle_array, bundle) {
     char* payload = NULL;
-    retcode_t rc = ta_mam_api_bundle_read(&mam, bundle, &payload);
-    if (rc != RC_OK) {
-      ret = SC_MAM_FAILED_RESPONSE;
-      ta_log_error("%s\n", "SC_MAM_FAILED_RESPONSE");
+    status_t read_ret = ta_mam_api_bundle_read(&mam, bundle, &payload);
+    if (read_ret == SC_TA_OOM) {
```

Comment:
What about other errors?

Reply:
I have listed the description for the special case here.

Reply:
Do we further use `SC_MAM_NO_PAYLOAD`?